lessons/ro/chapter_7.yaml

```
@@ -0,0 +1,272 @@
+- title: Capitolul 7 - Programare Orientata pe Obiecte
+  content_markdown: >
+    Exprimarea ideilor cu ajutorul functiilor este o metoda dovedita de a reprezenta
+    comportamente si date (Limbajul C o face de decenii!). 
```

```suggestion
    Exprimarea ideilor cu ajutorul funcțiilor este o metodă dovedită de a reprezenta
    comportamente și date (Limbajul C o face de decenii!). 
```

Please use Romanian diacritic letters.